doc/COMPILE.md

```diff
@@ -143,7 +143,7 @@ Note: You have to enclose a value with quotes `""` if it contains a semicolon (`
 E.g.:
 
 ```sh
-cmake -DPLUGINS="dump;resolver;yajl;list;spec" ..
+cmake -DPLUGINS="dump;resolver_fm_hpu_b;yajl;list;spec" ..
```

Comment:
Why? AFAIK `resolver` should default to `resolver_fm_hpu_b` anyway

Reply:
If this is the case (or should be), then it is not working on Mac. 

After running:
```sh
cmake -DPLUGINS="dump;resolver;yajl;list;spec"
```

I get the following error:
```text
CMake Error at src/plugins/CMakeLists.txt:81 (message):
  selected default resolver (resolver_fm_hpu_b) is not selected in PLUGINS,
  please change KDB_DEFAULT_RESOLVER or PLUGINS
```

Reply:
Okay, in that case keep the change and please open an issue so that we can investigate whether `-DPLUGINS=resolver` should work or not. It might just be that macOS defaults to another resolver variant and the default value for `KDB_DEFAULT_RESOLVER` is wrong.